### PR TITLE
Added an argument (flags) to support some pattern matching options 

### PR DESCRIPTION
**Pull request by spearmin10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Added an argument (flags) to support some pattern matching options ( `dotall`, `multiline`, `ignorecase` and `unicode`)

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by daryakoval — CHANGES_REQUESTED** [review]
Is the `caseless` argument is not in use anymore? If someone already using this script in his playbook with `caseless=true` argument, the output will be that same before and after the change?

File: Packs/MapPattern/Scripts/MapPattern/MapPattern.yml [hunk 1]
```diff
@@ -1,38 +1,15 @@
...
-  - last_match
-- description: '`demisto` context: Input . (single dot) on `From previous tasks` to
-    enable to extract the context data.'
-  name: context
+commonfields:
+  id: MapPattern
+  version: -1
+contentitemexportablefields:
+  contentitemfields:
+    fromServerVersion: ""
```
[the remark this answers is not in the document]
Comment (spearmin10) [reply Packs/MapPattern/Scripts/MapPattern/MapPattern.yml]: Removed those.

**Comment by daryakoval** [comment]
@spearmin10 Is the `caseless` argument is not in use anymore? If someone already using this script in his playbook with `caseless=true` argument, the output will be that same before and after the change?

**Comment by spearmin10** [comment]
> @spearmin10 Is the `caseless` argument is not in use anymore? If someone already using this script in his playbook with `caseless=true` argument, the output will be that same before and after the change?

`caseless` argument and `ignorecase` in `flags' are the same meaning. But `caseless` = True still work before the backward compatibility.

**Review by daryakoval — APPROVED** [review]

**Pull request merged**